_posts/2022/2022-09-19-use-before-and-after-methods-when-comparing-dates-and-times-in-rails.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: article
+title: Use before? and after? Methods When Comparing Dates and Times in Rails
+description: I can’t remember which is greater than or less than either.
+category: ruby
+image:
+  base: 2022/use-before-and-after-methods-when-comparing-dates-and-times-in-rails
+  alt: Clocks piled on top of each other
+  source: https://unsplash.com/photos/FlHdnPO6dlw
+  credit: Jon Tyson
+---
+
+Active Support contains many additions to the basic classes that form the standard libraries of Ruby. There are extensions to `String`s, `Hash`es, `Array`s and even the classes that manage `Date`s and `Time`s.
+
+In many cases if you’ve mostly been working inside Rails you might not realise that these are Rails-isms rather than Ruby-isms.
```

Comment:
```suggestion
If you’ve been mostly working inside Rails you might not realise that these are Rails-isms rather than Ruby-isms.
```

Not sure of the positioning of 'mostly' really...